copy-assets.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs-extra');
```

Comment:
Use this script to copy .csv and .json files from the `./src` directory to the TS built directory `./built`. This is needed because those files would be left behind from the TS compilation.